testnotes.md

```diff
@@ -0,0 +1,3 @@
+ret-fail.src/tgt.ll - This isn't testing equivalence, just that two functions are not equal ...
```

Comment:
This note is a bit mysterious to me. Remove or expand maybe?

Reply:
Not going to lie, I don't remember writing that note .... I've removed it for now.